variable/include/scipp/variable/creation.h

```diff
@@ -26,4 +26,7 @@ enum class SCIPP_VARIABLE_EXPORT FillValue {
 [[nodiscard]] SCIPP_VARIABLE_EXPORT Variable
 special_like(const VariableConstView &prototype, const FillValue &fill);
 
+Variable make_accumulant(const VariableConstView &var, Dim dim,
+                         const FillValue &init);
```

Comment:
I think we should avoid putting implementation details into "public" headers. A header such as `variable/operations_common.h` or a new one in `variable/` (not in `include/`) may be a better fit?

Reply:
Reverted